openpype/hosts/houdini/api/lib.py

```diff
@@ -562,19 +562,41 @@ def get_frame_data(node):
         dict: frame data for star, end and steps.
 
     """
+    if asset_data is None:
+        asset_data = {}
+
+    if log is None:
+        log = self.log
+
     data = {}
 
     if node.parm("trange") is None:
-
+        log.debug(
+            "Node has no 'trange' parameter: {}".format(node.path())
+        )
         return data
 
     if node.evalParm("trange") == 0:
-        self.log.debug("trange is 0")
-        return data
-
-    data["frameStart"] = node.evalParm("f1")
-    data["frameEnd"] = node.evalParm("f2")
-    data["steps"] = node.evalParm("f3")
+        data["frameStartHandle"] = hou.intFrame()
+        data["frameEndHandle"] = hou.intFrame()
+        data["byFrameStep"] = 1
+        data["handleStart"] = 0
+        data["handleEnd"] = 0
+        log.info(
+            "Node '{}' has 'Render current frame' set. \n"
+            "Asset Handles are ignored. \n"
+            "frameStart and frameEnd are set to the "
+            "current frame".format(node.path())
+        )
+    else:
+        data["frameStartHandle"] = int(node.evalParm("f1"))
+        data["frameEndHandle"] = int(node.evalParm("f2"))
+        data["byFrameStep"] = int(node.evalParm("f3"))
```

Comment:
Not too sure about these requiring to be `int` - as far as I know Houdini is capable of exporting from `1001.5` for example - even though it might not happen often so I guess we can force it out for `frameStartHandle` and `frameEndHandle`.

However, `byFrameStep` being `int` is **definitely** wrong. By doing so you're effectively breaking the capability to export "substeps" (sub frame data) from e.g. Alembic ROP. As far as I know that works by setting `byFrameStep` to e.g. `0.2` to have a sample ever 0.2 frames (1/0.2 = 5 samples per frame)

Reply:
oh, I got it! 
regarding, is there a middle ground so that it doesn't error in nuke ? 
is this sufficient ? 

```python
data["frameStartHandle"] = int(node.evalParm("f1"))
data["frameEndHandle"] = int(node.evalParm("f2"))
```

Reply:
> is this sufficient ? or should it be solved on nuke side ?

Admittedly - I'd definitely say it's **also** a bug on the Nuke loader side.

However, if current data storage gets more consistent for now we can force int for frame start/end I suppose. I actually believe the same is being done in Maya code currently.

Reply:
I checked Maya's code and yes. 
`frameStartHandle` and `frameEndHandle`  are always int! 
`frameStart` and `frameEnd` could be int or float 
- int when they are read from `asset["data"].get("frameStart")`
- float when they are read from `cmds.currentTime(query=True)`

Reply:
Thanks. Let's do this then:
```python
data["frameStartHandle"] = int(node.evalParm("f1"))
data["frameEndHandle"] = int(node.evalParm("f2"))
```
And leave by frame step as float (or whatever Houdini's value is)

Reply:
yeah... if we want to support frame start/ends being float then we need to change the assumptions in all the DCC loaders that expect it to be an int